sass/tji/_data.scss

```diff
@@ -0,0 +1,91 @@
+/*--------------------------------------------------------------
+>>> EXPLORE THE DATA TABLE OF CONTENTS:
+----------------------------------------------------------------
+# GENERAL
+# CHARTS
+
+/*--------------------------------------------------------------
+## CHARTS
+--------------------------------------------------------------*/
+
+#js-TJIChartView {
+  width: 100%;
+  padding: 2em;
+  background-color: #fbfbfb;
+  box-shadow: 2px 3px 6px rgba(64,64,64,0.5);
+  margin-left: 1em;
+  
+  h2 {
+    display: block;
+    width: 100%;
+    text-transform: uppercase;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    margin-bottom: 0.5em;
+    margin-top: 0;
+  }
+  
+  @media (max-width: 992px) {
+	  margin-left: 0;
+  }
+}
+
+.tji-data-count {
+	font-size: 18px;
+	margin-bottom: 1em;
+}
+
+.tji-chart {
+
+  border: 1px solid #ebebeb;
+  transition: all 0.25s;
+  
+  &:hover {
+    box-shadow: 0px 0px 20px rgba(64,64,64,0.15);
+    background-color: #fff;	    
+  }
+  
+  @media (min-width: 1500px) {
+	  -ms-flex-preferred-size: 33.33%;
+flex-basis: 33.33%;
+max-width: 33.33%;
+  }
+
+}
+
+#js-TJIfilters {
```

Comment:
I like to prefixes selectors for javascript only with js to keep a separation between functionality and styles. Feel free to add a class on the DOM element for styling and scoping.